src/app/containers/Footer/index.stories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Are we planning to make stories for containers? That seems like a strange thing to do to me as storybook is for seeing visual components so I'd assume we'd only make stories for presentational components?

Reply:
We certainly don't need one for this container at the very least. Let's remove it